### PR TITLE
Fix bug1518564-stop using bluebird

### PR DESCRIPTION
**Pull request by Biboswan**

### CODE REVIEW

**Comment by imbstack** [comment]
Closing and reopening to trigger a build.

**Comment by owlishDeveloper** [comment]
This needs to be rebased on master, I think

**Comment by owlishDeveloper** [comment]
Please rebase this branch on master

File: services/notify/src/irc.js [hunk 1]
```diff
@@ -1,11 +1,18 @@
...
-const Promise = require('bluebird');
-const irc = Promise.promisifyAll(require('irc-upd'));
+const util = require('util');
+const irc_upd = require('irc-upd');
 const assert = require('assert');
 const aws = require('aws-sdk');
 const _ = require('lodash');
 
 const MAX_RETRIES = 5;
+const irc = JSON.parse(JSON.stringify(irc_upd));
```
Comment (imbstack) [inline services/notify/src/irc.js]: Will this snippet actually work?

**Pull request closed without merging**